Update release process

askdba opened the pull request:
## Description

Update of the release process readme.

deepthi commented on doc/internal/ReleaseInstructions.md:
```diff
@@ -198,6 +198,44 @@ Install `gpg-agent` (needed below) e.g. on Ubuntu via: `sudo apt-get install gnu
+* Coordinate CNCF cross-posting Vitess Blog. 
+* Schedule and publish HubSpot Tweet on Vitess account. 
+* Update “.github/workflows/cluster_endtoend_upgrade.yml” workflow file on the main(master) branch with the new release.
+* Make a v0.9.0 tag for godoc / go modules (https://pkg.go.dev/vitess.io/vitess/?tab=versions). Find the tag of v9.0.0 release and use that to create a v0.9.0 tag. From your vitess.io/vitess checkout, do the following
```
It will be clearer to state at the beginning that we will use 9.0 as an example throughout and the actual numbers need to change each release.

deepthi commented on doc/internal/ReleaseInstructions.md:
```diff
@@ -214,17 +214,23 @@ Create the `settings.xml` in the `$HOME/.m2/` directory as described in their [i
 
 ### Post Release Steps
 * Announce new release in Vitess Slack #general channel. 
 * Create a new [PR](https://github.com/vitessio/website/pull/670) for Vitess Blog.
```
PR should be created and approved pre-release so that as soon as release is out we can merge and  the blog post.

deepthi reviewed (commented):
Quite a few things still need to change:
1. The Versioning section needs to be re-written to be in line with VEPs which live here https://github.com/vitessio/enhancements/tree/master/veps
2. Docker images section should be removed. Docker images are built automatically on dockerhub.

@harshit-gangal can you review the Java section? Is it still accureate?

askdba commented:
> Another previously undocumented step is that I have been manually running https://github.com/vitessio/vitess/blob/master/helm/release.sh once the `base` Docker image is live.

@derekperkins can you point out where this step should go exactly?

derekperkins commented:
It can go at the very end. The only prerequisite is that the vitess/base image has been tagged

frouioui commented:
@askdba, I have added my takes: I changed the release creation images and added a bit more details for each step. Let me know if the pull request is ready for review

askdba commented:
> @askdba, I have added my takes: I changed the release creation images and added a bit more details for each step. Let me know if the pull request is ready for review

Asking @harshit-gangal to check Java sections.

askdba commented:
> It can go at the very end. The only prerequisite is that the vitess/base image has been tagged

Done

harshit-gangal commented:
> > @askdba, I have added my takes: I changed the release creation images and added a bit more details for each step. Let me know if the pull request is ready for review
> 
> Asking @harshit-gangal to check Java sections.

nothing changed in Java section. Looks good.
